src/main/cpp/xmlhw/MechanismDefn.cpp

```diff
@@ -164,12 +152,4 @@ void MechanismDefn::ParseXML
             Logger::GetLogger()->LogError( string("MechanismDefn"), msg );
         }
     }
-
-
-    // create instance
-    if ( !hasError )
-    {
-        MechanismFactory* factory =  MechanismFactory::GetMechanismFactory();
-        factory->CreateIMechanism( type, motors, servos, digitalInputs, canCoder );

```

Comment:
Can't delete the whole line, need to just remove servos from the call sequence otherwise we won't create any mechanisms